deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -29,8 +29,13 @@ install_fluentd: true
 # and 4.1.1 for other distros.
 td_agent_version: ""
 
-splunk_fluentd_config: /etc/otel/collector/fluentd/fluent.conf
+splunk_fluentd_config: ""
 splunk_fluentd_config_source: ""
 
-agent_bundle_dir: /usr/lib/splunk-otel-collector/agent-bundle
-splunk_collectd_dir: /usr/lib/splunk-otel-collector/agent-bundle/run/collectd
+agent_bundle_dir: ""
+splunk_collectd_dir: ""
+
+package_stage: release
+
+msi_name: splunk-otel-collector-{{splunk_otel_collector_version}}-amd64.msi
+win_base_url: https://dl.signalfx.com
```

Comment:
Same for these two. No need to make it configurable at this point

Reply:
msi_name should be configurable to fetch latest version.

Reply:
Why? Can't the version be configurable with `splunk_otel_collector_version`?

Reply:
Hello @dmitryax, When we set msi_name using set_fact it's only fetching the version as "latest" from defaults/main.yml. The latest version we are fetching from dl.signalfx.com is not overriding the default value "latest". We can give the name of msi directly in the "URL" but lint test is failing due to error line too long.

Reply:
> We can give the name of msi directly in the "URL" but lint test is failing due to error line too long.

You can just break down the line as done here for example https://github.com/signalfx/splunk-otel-collector/blob/5f2b860291a4872e868961cab809d629e724bd69/deployments/ansible/roles/collector/tasks/yum_install_fluentd.yml#L12-L13